### PR TITLE
[Bugfix] Use default arguments in PHP 7+

### PR DESCRIPTION
**Pull request by peter-gribanov**
Fix for #14

Impossible use NULL if type of argument is declared.
https://3v4l.org/pvHki
https://3v4l.org/CNgtY

### CODE REVIEW

**Comment by TomasVotruba** [comment]
I think this can be merged.

Ping @jupeter

File: README.md [hunk 1]
```diff
@@ -171,22 +171,40 @@ function paintCar(&$car) {
...
+{
+    // ...
 }
-
 ```
 
-**Good**:
+**Not bad:**
+
+This opinion is understandable than the previous version, but it better controls the value of the variable.
```
Comment (alexisLefebvre) [inline README.md]: Should it be

> This opinion is **more** understandable than the previous version

?

Comment (peter-gribanov) [reply README.md]: I fixed it. Thanks

**Comment by TomasVotruba** [comment]
Merging, thank you for the work

**Pull request merged**